src/blocks/Card.scss

```diff
@@ -0,0 +1,133 @@
+.card {
+  /* Component Variables */
+  --card-spacing-base: var(--bloom-s6);
+  --card-spacing-sm: var(--bloom-s4);
+  --card-spacing-md: var(--bloom-s8);
+  --card-spacing-lg: var(--bloom-s12); 
+
+  --card-spacing: var(--card-spacing-base);
+
+  --card-background-color: var(--bloom-color-white);
+  --card-border-radius: var(--bloom-rounded-lg);
+  --card-border-width: var(--bloom-border-1);
+  --card-border-color: var(--bloom-color-gray-450);
+  --card-divider-width: var(--card-border-width);
+  --card-divider-color: var(--card-border-color);
+
+  --card-content-padding-block: var(--card-spacing);
+  --card-content-padding-inline-desktop: var(--card-content-padding-block);
+  --card-content-padding-inline-mobile: var(--card-content-padding-block);
+  
+  --card-header-padding-block: var(--card-spacing);
+  --card-header-padding-inline-desktop: var(--card-header-padding-block);
+  --card-header-padding-inline-mobile: var(--card-header-padding-block);
+
+  --card-footer-background-color: var(--card-background-color);
+
+  /* Responsive Settings */
+
+  --card-header-padding-inline: var(--card-header-padding-inline-desktop);
+  --card-content-padding-inline: var(--card-content-padding-inline-desktop);
+
+  @media (max-width: 640px) { /* small screen */
```

Comment:
Do we / could we potentially have 640 as a var somewhere?

Reply:
Yeah, we do have screen values as tokens in the CSS, but unfortunately media queries are one area where native CSS variables aren't supposed, so we'll have to fallback to something like Sass variables (which is what happens in ui-components, but it's not currently in ui-seeds). I'll file a separate ticket to investigate a good solution.